language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -130,18 +130,28 @@ packageNameText pkgId mbPkgIdent = maybe (unPackageId pkgId) unPackageName mbPkg
 
 newtype Scope = Scope {unscope :: String}
 newtype Dependency = Dependency {undependency :: T.Text}  deriving (Eq, Ord)
+newtype SdkVersion = SdkVersion String
 
 -- Gives the scope 'foo' given a directory path like '/path/to/foo'.
 -- The scope 'foo' is the '@foo' part of a package name in an import
 -- declaration.
 scopeOfScopeDir :: FilePath -> Scope
 scopeOfScopeDir = Scope . takeFileName
 
+data Daml2TsParams = Daml2TsParams
+  { opts :: Options  -- cli args
+  , pm :: Map.Map PackageId (Maybe PackageName, Package)
```

Comment:
maybe just call this `packageMap`? `pm` seems a bit too generic and having worked with LLVM, I always things of `PassManager`.

Reply:
Going with `pkgMap` (consistency with other parameters).